### DISCUSSION TITLE
questions about build

### DISCUSSION THREAD

**Discussion by iamqiz** [Tampermonkey/tampermonkey-editors#3] [General]
i build this extension according to readme, but I didn't succeed in making the built editor extension work (when click icon,it only open vscode.dev, but did not list script from tampermonkey and no error /message in console for service worker)
here are my question about the extension:
1. Do I have to use packed extension (chrome-999.crx) ? can I use the Unpacked one (release/out/rel  or chrome-999.zip) ?
2. can the extension that built in ubuntu be used in chrome for windows 10?

thanks !

**Comment by jerone** [comment]
Not sure if you really want to make a build yourself of just use this functionality, but you can install the browser extension:
* https://chromewebstore.google.com/detail/tampermonkey-editors/lieodnapokbjkkdkhdljlllmgkmdokcm?pli=1
* https://addons.mozilla.org/en-US/firefox/addon/tampermonkey-editors/

_(Might be good to mention these links on the readme.)_

**Comment by derjanb** [comment]
> can I use the Unpacked one (release/out/rel or chrome-999.zip) ?

Yes, but you have to patch the extension ID of this extension (instead of `kjmbknaomholdmpocgplbkgmjdnidinh`) into your unpacked Tampermonkey. 
Please note that the official Tampermonkey extensions only accepts requests from the official Tampermonkey Editors extensions.

> can the extension that built in ubuntu be used in chrome for windows 10?

Yes.

**Reply by iamqiz** [reply]
thanks! it work ！

**Comment by derjanb** [comment]
> (Might be good to mention these links on the readme.)

👍
